resources/qml/TimelineView.qml

```diff
@@ -420,6 +422,42 @@ Item {
         angle: 90
     }
 
+    ParticleSystem {
```

Comment:
Why do you need multiple particle systems?

Reply:
~~Otherwise both kinds of particles get emitted whenever you emit something~~ I just realized that you can work around that with particle groups.

Reply:
Right, but all the particles were in the same logical group, so they all got emitted from either emitter. Splitting the particles into groups fixes that issue though.